debian.aws/config/OVERRIDES

```diff
@@ -0,0 +1,234 @@
+#
+# Disable various "net" modules which we don't use.
+#
+CONFIG_6LOWPAN=n
+CONFIG_6LOWPAN_NHC=n
+CONFIG_BT=n
+CONFIG_CAIF=n
+CONFIG_HAMRADIO=n
+CONFIG_IEEE802154=n
+CONFIG_NET_9P=n
+CONFIG_NET_DSA=n
+CONFIG_NFC=n
+CONFIG_RFKILL=n
+CONFIG_WIMAX=n
+CONFIG_WIRELESS=n
+
+#
+# Disable various "drivers" modules which we don't use.
+#
+CONFIG_ACCESSIBILITY=n
+CONFIG_ATA=n
+CONFIG_ATM_DRIVERS=n
+CONFIG_AUXDISPLAY=n
+CONFIG_BCMA=n
+CONFIG_CONNECTOR=n
+CONFIG_CRYPTO_HW=n
+CONFIG_DAX=n
+CONFIG_EISA=n
+CONFIG_EXTCON=n
+CONFIG_FPGA=n
+CONFIG_FSI=n
+CONFIG_GNSS=n
+CONFIG_GPIOLIB=n
+CONFIG_GREYBUS=n
+CONFIG_HSI=n
+CONFIG_I3C=n
+CONFIG_IDE=n
+CONFIG_IIO=n
+CONFIG_INFINIBAND=n
+CONFIG_INTERCONNECT=n
+CONFIG_IPACK_BUS=n
+CONFIG_LIBNVDIMM=n
+CONFIG_MAILBOX=n
+CONFIG_MCB=n
+CONFIG_MD=n
+CONFIG_MEDIA_SUPPORT=n
+CONFIG_MEMSTICK=n
+CONFIG_MMC=n
+CONFIG_MTD=n
+CONFIG_NEW_LEDS=n
+CONFIG_NTB=n
+CONFIG_NVM=n
+CONFIG_NVMEM=n
+CONFIG_OF=n
+CONFIG_PARPORT=n
+CONFIG_PARPORT_PANEL=n
+CONFIG_PCCARD=n
+CONFIG_PINCTRL=n
+CONFIG_PMIC_OPREGION=n
+CONFIG_PM_DEVFREQ=n
+CONFIG_POWERCAP=n
+CONFIG_PPS=n
+CONFIG_RAPIDIO=n
+CONFIG_REGULATOR=n
+CONFIG_RESET_CONTROLLER=n
+CONFIG_RTC_CLASS=n
+CONFIG_SFI=n
+CONFIG_SIOX=n
+CONFIG_SLIMBUS=n
+CONFIG_SOUNDWIRE=n
+CONFIG_SPI=n
+CONFIG_SPMI=n
+CONFIG_STAGING=n
+CONFIG_THERMAL=n
+CONFIG_THUNDERBOLT=n
+CONFIG_UIO=n
+CONFIG_USB_SUPPORT=n
+CONFIG_VFIO=n
+CONFIG_W1=n
+
+#
+# Disable various filesystem which we don't use.
+#
+CONFIG_9P_FS=n
+CONFIG_ADFS_FS=n
+CONFIG_AFFS_FS=n
+CONFIG_AFS_FS=n
+CONFIG_AUFS_FS=n
+CONFIG_BEFS_FS=n
+CONFIG_BFS_FS=n
+CONFIG_BTRFS_FS=n
+CONFIG_CEPH_FS=n
+CONFIG_CODA_FS=n
+CONFIG_ECRYPT_FS=n
+CONFIG_EFS_FS=n
+CONFIG_EROFS_FS=n
+CONFIG_EXFAT_FS=n
+CONFIG_EXT2_FS=n
+CONFIG_EXT3_FS=n
+CONFIG_F2FS_FS=n
+CONFIG_FAT_FS=n
+CONFIG_GFS2_FS=n
+CONFIG_HFSPLUS_FS=n
+CONFIG_HFS_FS=n
+CONFIG_HPFS_FS=n
+CONFIG_JFFS2_FS=n
+CONFIG_JFS_FS=n
+CONFIG_MINIX_FS=n
+CONFIG_MSDOS_FS=n
+CONFIG_NILFS2_FS=n
+CONFIG_OCFS2_FS=n
+CONFIG_OMFS_FS=n
+CONFIG_ORANGEFS_FS=n
+CONFIG_QNX4FS_FS=n
+CONFIG_QNX6FS_FS=n
+CONFIG_REISERFS_FS=n
+CONFIG_REISERFS_FS=n
+CONFIG_ROMFS_FS=n
+CONFIG_SYSV_FS=n
+CONFIG_UBIFS_FS=n
+CONFIG_UDF_FS=n
+CONFIG_UFS_FS=n
+CONFIG_VFAT_FS=n
+CONFIG_VXFS_FS=n
+CONFIG_XFS_FS=n
+
+#
+# Disable other misc. modules which we don't use.
+#
+CONFIG_AGP=n
+CONFIG_ANDROID=n
+CONFIG_AQUANTIA_PHY=n
+CONFIG_ATALK=n
+CONFIG_BATMAN_ADV=n
+CONFIG_BCACHE=n
+CONFIG_BE2ISCSI=n
+CONFIG_BE2NET=n
+CONFIG_BLK_DEV_PCIESSD_MTIP32XX=n
+CONFIG_BLK_DEV_RSXX=n
+CONFIG_BNA=n
+CONFIG_BNX2=n
+CONFIG_BNX2X=n
+CONFIG_BNXT=n
+CONFIG_BROADCOM_PHY=n
+CONFIG_CASSINI=n
+CONFIG_CDROM=n
+CONFIG_CEPH_LIB=n
+CONFIG_CIFS=n
+CONFIG_CRAMFS=n
+CONFIG_CYCLADES=n
+CONFIG_DRM=n
+CONFIG_ENIC=n
+CONFIG_FM10K=n
+CONFIG_FORCEDETH=n
+CONFIG_GENWQE=n
+CONFIG_GVE=n
+CONFIG_HABANA_AI=n
+CONFIG_HID=n
+CONFIG_HIO=n
+CONFIG_HWMON=n
+CONFIG_IAVF=n
+CONFIG_ICE=n
+CONFIG_INFINIBAND=n
+CONFIG_INTEL_IOATDMA=n
+CONFIG_INTEL_MEI=n
+CONFIG_IONIC=n
+CONFIG_JME=n
+CONFIG_KVM=n
```

Comment:
Don't we need KVM for Docker?

Reply:
I don't think so.. but let me fine out what test suite I can run to stress the docker stuff.. that way I can verify.

I think KVM is creating virtual machines on the host.. which we don't do.. since a container is different than a virtual machine.. my intention was not to modify any of the container stuff, but turn off the hypervisor functionality.